content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -244,10 +245,11 @@ try {
 }
 ```
 
-The interactive transaction API has a 2nd parameter with the following configuration options:
+The interactive transaction API has a second parameter with the following configuration options:
```

Comment:
Seems like these are all optional and have a default. Might be worth mentioning here:
```suggestion
The interactive transaction API has a second parameter with the following optional configuration options:
```